flutter_appauth/ios/Classes/OIDExternalUserAgentIOSNoSSO.h

```diff
@@ -0,0 +1,46 @@
+/*! @file OIDExternalUserAgentIOS.h
```

Comment:
This doesn't match the name of the file. Could you rename it appropriately and include an attribution on what it's based on? This could perhaps include a link to the local of the file in the AppAuth iOS GitHub repo